app/README.md

```diff
@@ -71,25 +71,22 @@ The filter `setid` is used in `/app/.gitattributes` so that it replaces your app
 #### Install the app to your Jira instance
 
 1. Build your app by running `yarn build` in **/app/jenkins-for-jira-ui**.
-2. Deploy your app by running `forge deploy` in **/app**. See [Common Issues](#common-issues) if unsuccessful.
+2. Deploy your app by running `forge deploy --no-verify` in **/app**. See [Common Issues](#common-issues) if unsuccessful.
 3. Install your app to your Jira site by running `forge install` in **/app**.
 
 Follow the instructions in this setup wizard. Enter the hostname for your Jira site (e.g. `<your-site-name>.atlassian.net`).
 
 > NOTE: This is the *only* step that needs to be repeated to install on different Jira sites. Do not repeat the steps above `forge deploy`. To run your app, follow the steps below.
 
 ## Running the app
-To run the app, make sure you have installed the app on your site as described above. You'll also need to your `ngrok-config-path` so you can create a local tunnel. Sign up to [ngrok](https://ngrok.com/) if you have not done so. To do this:
```

Comment:
Why has this been removed? 
Just having an empty title doesn't look right.

Reply:
https://developer.atlassian.com/platform/forge/changelog/#CHANGE-2159